src/js/components/FilterButtons.js

```diff
@@ -62,19 +62,21 @@ class FilterButtons extends React.Component {
 }
 
 FilterButtons.defaultProps = {
-  renderButtonContent: function (title) {return title; },
-  onFilterChange: function () {}
+  inverseStyle: false,
+  onFilterChange: function () {},
+  renderButtonContent: function (title) {return title; }
```

Comment:
i actually don't think we have a linting for that rule. there are specific cases where we don't do it. i think this is good 👍